starter-code/styles/base.css

```diff
@@ -11,6 +10,12 @@ hr {
   width: 80%;
 }
 
+/*IMAGE STYLES*/
+/*=======================================*/
+img {
```

Comment:
By making width 100%, it made the width 100% of the page, but the TODO specifies that images never exceed their containers width. How would we do that?